ui/.env.development

```diff
@@ -1,4 +1,4 @@
 REACT_APP_BASENAME="/MAAS/r"
-REACT_APP_BASE_URL="http://localhost:5240/"
-REACT_APP_MAAS_URL="http://localhost:5240/MAAS"
-REACT_APP_WEBSOCKET_URL="ws://localhost:5240/MAAS/ws"
+REACT_APP_BASE_URL="http://karura.internal:5240/"
+REACT_APP_MAAS_URL="http://karura.internal:5240/MAAS"
+REACT_APP_WEBSOCKET_URL="ws://karura.internal:5240/MAAS/ws"
```

Comment:
Is it worth updating the README to mention that by default maas-ui will connect to karura in development?

Reply:
Good idea.